templates/components/api-server/template/server/middleware.json

```diff
@@ -3,7 +3,17 @@
     "loopback#favicon": {}
   },
   "initial": {
-    "compression": {}
+    "compression": {},
+    "cors": {
+      "params": {
+        "origin": true,
+        "credentials": true,
+        "exposedHeaders": true,
+        "methods": "GET,POST,PUT,PATCH,DELETE,OPTIONS",
+        "allowedHeaders": "Content-Type,Authorization",
+        "maxAge": 180
```

Comment:
Maybe we should set `maxAge` to a higher number as we don't change `CORS` policies dynamically.